src/smartactions/ScheduledOrders.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {IERC7579Execution} from "modulekit/Accounts.sol";
+import {SchedulingBase} from "./SchedulingBase.sol";
+import {UniswapV3Integration} from "modulekit/Integrations.sol";
+import {IERC20} from "forge-std/interfaces/IERC20.sol";
+
+abstract contract ScheduledOrders is SchedulingBase {
+    function executeOrder(uint128 jobId) external override canExecute(jobId) {
+        ExecutionConfig storage executionConfig = _executionLog[msg.sender][
+            jobId
+        ];
+
+        // decode from execution tokenIn, tokenOut and amount in
+        (
+            address tokenIn,
+            address tokenOut,
+            uint256 amountIn,
+            uint160 sqrtPriceLimitX96
```

Comment:
sqrt is part of the `bytes executionData` as well as the `ExecutionConfig` struct? surely this is a mistake

Reply:
Yes, should be removed from `ExecutionConfig `